apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -499,13 +502,21 @@ static void tc_task_task_reparent(void)
 */
 static void tc_task_task_init(void)
 {
+	int ret = 0;
+
 	task_init_flag = false;
 
+	sem_init(&task_sem, 0, 0);
+
 	TC_ASSERT_EQ("task_init", ioctl(tc_get_drvfd(), TESTIOC_TASK_INIT_TEST, (unsigned long)test_task_entry), OK);
 
 	usleep(1);
 
-	TC_ASSERT_EQ("task_init", task_init_flag, true);
+	ret = sem_wait(&task_sem);
+	TC_ASSERT_EQ_CLEANUP("task_init", ret, OK, sem_destroy(&task_sem));
```

Comment:
This macro's 1st param should be changed to `sem_wait` because this `ret` is from `sem_wait`.
You don't need to worry because macro will print which TC is.

Reply:
Could you check what Jeongchan said?
```
TC_ASSERT_EQ_CLEANUP("sem_wait", ....);
```

Reply:
Please check above and let me know.

Reply:
modified